satellite/marketing/database.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package marketing
```

Comment:
I am not sure this is a good package name, especially with its location being in the satellite.
But i dont have a good idea how to do it better either. Just sounds wrong to me :)

Reply:
I think once we add more functionality to it, it will become more clear how we should name this package. Right now, the purpose of this package is so that marketing team can change configuration on the satellite for marketing purpose

Reply:
What configuration would that be? the amount of free credits?

Reply:
We will have a Open Source Partner Program and it will have configuration that need to be exposed to the marketing team to be able to change it. Not sure what exact configuration we will have at the moment